app/models/concerns/settings_concern.rb

```diff
@@ -23,7 +23,7 @@ def enabled_component_ids
 
   # Disable/Enable components
   #
-  # @params [Array<String>] the ids of all the enabled components
+  # @param ids [Array<String>] the ids of all the enabled components
```

Comment:
That will cause an `unknown parameter name` warning. See the last comment at http://stackoverflow.com/questions/11311973/warning-in-yard-with-param-tag